homeassistant/components/plugwise/gateway.py

```diff
@@ -42,16 +46,17 @@ async def async_setup_entry_gw(hass: HomeAssistant, entry: ConfigEntry) -> bool:
 
     try:
         connected = await api.connect()
-    except InvalidAuthentication:
-        LOGGER.error("Invalid username or Smile ID")
-        return False
-    except (ClientConnectionError, PlugwiseException) as err:
+    except ConnectionFailedError as err:
+        raise ConfigEntryNotReady("Failed to connect to the Plugwise Smile") from err
```

Comment:
The name was remove/lost, was this intentional?

Reply:
Yes.